js/client-battle.js

```diff
@@ -951,14 +952,72 @@
 			case 'move':
 				var move = Tools.getMove(thing);
 				if (!move) return;
+				// Declare variables we are going to need.
```

Comment:
My personal rule for comments is: Never document _what_ the code does (programmers know how to read code and don't need to be told that), document _why_ the code does it.